### ISSUE TITLE
Style of the Message Input Field

### ISSUE THREAD

**Issue by Eladnarlea** [humhub/mail#294]
Please also adjust the Style of the Message Input Field.

inactive style:

<img width="1043" alt="Bildschirmfoto 2023-01-26 um 08 59 45" src="https://user-images.githubusercontent.com/105652258/214785364-6b667c26-c159-451e-a4d2-1d8ceb09b0c0.png">

active style:

<img width="1043" alt="Bildschirmfoto 2023-01-26 um 08 59 56" src="https://user-images.githubusercontent.com/105652258/214785355-8ff1b15d-0cfa-4bdb-8e74-44b5f419495f.png">

**Comment by luke-** [comment]
@Eladnarlea Is it about the font colour?

Currently it should be the standard form style, which is also used in all other forms e.g. in comment fields, post form.

If we want to change form inputs, we should do this globally.

**Comment by Eladnarlea** [comment]
No, this is about the global style change. I wanted to implement a new version with small changes of it here first, so we could then adjust the other ones, too. I changed the title of the Issue to "Future change of style of..."
